src/js/site/highlightURLHash.test.js

```diff
@@ -0,0 +1,74 @@
+import { highlightURLHash, addHighlightClassIfInUrl } from './highlightURLHash';
+
+document.body.innerHTML = '<div id="overallSuccesRate-definition"></div>';
```

Comment:
This should not be here as it will be in every test run after that. Suggesting that you put it in the `beforeEach` and perform a removal in the `afterEach`